rfc6962-bis.xml

```diff
@@ -394,7 +394,7 @@ certificate. Logs MUST also reject precertificates that are not valid DER encode
     } PrecertChainEntryV2;</artwork>
         </figure>
         <t>
-          Logs SHOULD limit the length of chain they will accept.
+          Logs SHOULD limit the length of chain they will accept (which is a part of the log's metadata).
```

Comment:
The grammar seems a bit clunky there.  How about:
"Logs SHOULD limit the length of chain they will accept.  The maximum chain length is specified in the log's metadata."